client/landing/stepper/declarative-flow/flows/new-hosted-site-flow/new-hosted-site-flow.ts

```diff
@@ -120,22 +121,36 @@ const hosting: FlowV2< typeof initialize > = {
 			}
 		};
 
+		const scheduleSiteCreation = () => {
+			setPendingAction( () =>
+				createSite( {
+					siteVisibility: Site.Visibility.PublicNotIndexed,
+					useThemeHeadstart: false,
+				} )
+			);
+			return navigate( STEPS.PROCESSING.slug );
+		};
```

Comment:
Is there a reason to make this a callback rather than inlining it? Just trying to understand.